src/main/resources/db/migration/data/R__5_9__OFFENDER_VISIT_VISITORS.sql

```diff
@@ -1,11 +1,49 @@
+INSERT INTO OFFENDER_VISIT_VISITORS (OFFENDER_VISIT_VISITOR_ID, OFFENDER_VISIT_ID, OFFENDER_BOOK_ID, EVENT_ID, EVENT_STATUS, GROUP_LEADER_FLAG, EVENT_OUTCOME, OUTCOME_REASON_CODE, COMMENT_TEXT) VALUES (3125, -25, -3, 125, 'SCH', 'N', NULL, NULL, NULL);
+INSERT INTO OFFENDER_VISIT_VISITORS (OFFENDER_VISIT_VISITOR_ID, OFFENDER_VISIT_ID, OFFENDER_BOOK_ID, EVENT_ID, EVENT_STATUS, GROUP_LEADER_FLAG, EVENT_OUTCOME, OUTCOME_REASON_CODE, COMMENT_TEXT) VALUES (3128, -28, -5, 128, 'SCH', 'N', NULL, NULL, NULL);
+INSERT INTO OFFENDER_VISIT_VISITORS (OFFENDER_VISIT_VISITOR_ID, OFFENDER_VISIT_ID, OFFENDER_BOOK_ID, EVENT_ID, EVENT_STATUS, GROUP_LEADER_FLAG, EVENT_OUTCOME, OUTCOME_REASON_CODE, COMMENT_TEXT) VALUES (3130, -30, -6, 130, 'SCH', 'N', NULL, NULL, NULL);
+INSERT INTO OFFENDER_VISIT_VISITORS (OFFENDER_VISIT_VISITOR_ID, OFFENDER_VISIT_ID, OFFENDER_BOOK_ID, EVENT_ID, EVENT_STATUS, GROUP_LEADER_FLAG, EVENT_OUTCOME, OUTCOME_REASON_CODE, COMMENT_TEXT) VALUES (3131, -31, -6, 131, 'SCH', 'N', NULL, NULL, NULL);
+INSERT INTO OFFENDER_VISIT_VISITORS (OFFENDER_VISIT_VISITOR_ID, OFFENDER_VISIT_ID, OFFENDER_BOOK_ID, EVENT_ID, EVENT_STATUS, GROUP_LEADER_FLAG, EVENT_OUTCOME, OUTCOME_REASON_CODE, COMMENT_TEXT) VALUES (3132, -32, -6, 132, 'SCH', 'N', NULL, NULL, NULL);
+INSERT INTO OFFENDER_VISIT_VISITORS (OFFENDER_VISIT_VISITOR_ID, OFFENDER_VISIT_ID, OFFENDER_BOOK_ID, EVENT_ID, EVENT_STATUS, GROUP_LEADER_FLAG, EVENT_OUTCOME, OUTCOME_REASON_CODE, COMMENT_TEXT) VALUES (3133, -33, -6, 133, 'SCH', 'N', NULL, NULL, NULL);
+INSERT INTO OFFENDER_VISIT_VISITORS (OFFENDER_VISIT_VISITOR_ID, OFFENDER_VISIT_ID, OFFENDER_BOOK_ID, EVENT_ID, EVENT_STATUS, GROUP_LEADER_FLAG, EVENT_OUTCOME, OUTCOME_REASON_CODE, COMMENT_TEXT) VALUES (3134, -34, -6, 134, 'SCH', 'N', NULL, NULL, NULL);
+INSERT INTO OFFENDER_VISIT_VISITORS (OFFENDER_VISIT_VISITOR_ID, OFFENDER_VISIT_ID, OFFENDER_BOOK_ID, EVENT_ID, EVENT_STATUS, GROUP_LEADER_FLAG, EVENT_OUTCOME, OUTCOME_REASON_CODE, COMMENT_TEXT) VALUES (3135, -35, -6, 135, 'SCH', 'N', NULL, NULL, NULL);
+INSERT INTO OFFENDER_VISIT_VISITORS (OFFENDER_VISIT_VISITOR_ID, OFFENDER_VISIT_ID, OFFENDER_BOOK_ID, EVENT_ID, EVENT_STATUS, GROUP_LEADER_FLAG, EVENT_OUTCOME, OUTCOME_REASON_CODE, COMMENT_TEXT) VALUES (3136, -36, -6, 136, 'SCH', 'N', NULL, NULL, NULL);
+INSERT INTO OFFENDER_VISIT_VISITORS (OFFENDER_VISIT_VISITOR_ID, OFFENDER_VISIT_ID, OFFENDER_BOOK_ID, EVENT_ID, EVENT_STATUS, GROUP_LEADER_FLAG, EVENT_OUTCOME, OUTCOME_REASON_CODE, COMMENT_TEXT) VALUES (3137, -37, -6, 137, 'SCH', 'N', NULL, NULL, NULL);
 
+INSERT INTO OFFENDER_VISIT_VISITORS (OFFENDER_VISIT_ID,PERSON_ID,GROUP_LEADER_FLAG,OFFENDER_VISIT_VISITOR_ID,EVENT_OUTCOME,OFFENDER_BOOK_ID,EVENT_ID,CREATE_USER_ID,EVENT_STATUS) VALUES (-15,-1,'Y',-1,'ABS',null,400001,'ME','CANC');
```

Comment:
inserting the `EVENT_STATUS` here shouldn't be required, but still needed as other repository code still grabs the values from the lead visitor instead.  Will have to be tidied up at a later date...